word_counter.rb

```diff
@@ -0,0 +1,79 @@
+require 'pragmatic_segmenter'
+# how do you include a gem such as this so that the program will run on a given computer?
+
+def word_counter
```

Comment:
> If this is a gem, then you should have a Gemfile and .gemspec file to keep track of all the gems that you'll use.

- Are you asking if all 80 lines of code are to be used as a gem? (I don't really know the answer to that) OR are you asking if line 1 is a gem? Line 1, "require 'pragmatic segmenter'" is a gem.

> I suggest exposing the core functionality you have in a rake task so that it's easy to run as a script from the command line.

- I've never done this before but I'll give it a shot...

> I know this is just a code sample, but please add some rspec tests. It helps other people understand what your code does, it shows that your code works, and it shows that your work is clean and well-rounded.

- I'll give this a shot.

> Move the core logic in the gem to a lib folder.

- I'm not sure what you mean by this.